source/element-parameters.md

```diff
@@ -266,3 +297,135 @@ BACKWARDS           # Injected particle propagates in reverse direction.
 ```
 
 See the [](#s:forking) chapter for more details.
+
+%---------------------------------------------------------------------------------------------------
+(s:init.particle.params)=
+## InitialParticleP Initial Particle Coordinates Parameters
+
+In Construction...
+
+%---------------------------------------------------------------------------------------------------
+(s:mag.mult.params)=
+## MagneticMultipoleP Magnetic Multipole Parameters
+
+The `MagneticMultipoleP` parameter group describes magnetic multipoles associated with the lattice
+element. For a multipole of order `N`, the magnetic field {math}`(B_x, B_y)`
+at a point {math}`\bf r` in terms of the points polar coordinates {math}`(r, \theta)` is
+```{math}
+:label: bbmult
+
+B_x({\bf r}) + i \, B_y({\bf r}) = 
+\frac{1}{N!}(B_n + B_s) \, e^{-i(N+1)T} \, e^{iN\theta} \, r^N
```

Comment:
I think line 318 missing a complex unit i before `B_s`?  Otherwise, the normal and skew coefficients are treated identically.

Reply:
Correct. I will change.